inclusive-menu-button.js

```diff
@@ -4,7 +4,26 @@
   'use strict'
 
   // Constructor
-  function MenuButton (button) {
+  function MenuButton (button, userSettings) {
+    // Error if the userSettings argument isn't an object
+    if (userSettings !== undefined && typeof userSettings !== 'object') {
+      throw new Error('MenuButton\'s second argument is expected as an object or undefined')
+    }
+
+    // The default settings
+    this.settings = {
+      radios: false
```

Comment:
I’m not too sure about the naming here to be faire. Maybe something like `checkable: <bool>` would be more obvious?